hawkbit-mgmt-resource/src/test/java/org/eclipse/hawkbit/mgmt/rest/resource/MgmtDistributionSetTypeResourceTest.java

```diff
@@ -554,7 +555,7 @@ public void invalidRequestsOnDistributionSetTypesResource() throws Exception {
                 .contentType(MediaType.APPLICATION_JSON)).andDo(MockMvcResultPrinter.print())
                 .andExpect(status().isBadRequest());
 
-        final DistributionSetType missingName = entityFactory.generateDistributionSetType("test123", null, "Desc123");
+        final DistributionSetType missingName = new JpaDistributionSetType("test123", null, "Desc123");
```

Comment:
just thought that actually the mgmt-resource module shouldn't know about `JPA` so this was actually hided behind the `entityFactory`, now this test depends on a JPA implementation would it make sense to not use the Jpa-Entity instead a JSON string for the mgmt-resource tests